### ISSUE TITLE
fix broken VmetFixers

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/fontbakery#760]

**Comment by felipesanches** [comment]
This is the current error (before disabling the broken fixer):

```
note: FAIL: bakery_lint/fonttests/test_ttf.py: TTFTestCase.test_metrics_descents_equal_bbox():  Check that descents values are same as min glyph point : [Coiny-Regular2911.ttf] ascents differ to minimum value: -546
<method = '<class 'bakery_cli.fixers.VmetFixer'>'>
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_cli/pipe/fontlint.py", line 61, in run
    data = run_set(op.join(self.builddir, processedfile), 'result', apply_fix=True)
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_lint/__init__.py", line 27, in run_set
    return run_suite(tests_suite)
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_lint/base.py", line 333, in run_suite
    runner.run(suite)
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_lint/base.py", line 254, in run
    test(result)
  File "/usr/lib/python2.7/unittest/suite.py", line 70, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python2.7/unittest/suite.py", line 108, in run
    test(result)
  File "/usr/lib/python2.7/unittest/case.py", line 395, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python2.7/unittest/case.py", line 335, in run
    result.addFailure(self, sys.exc_info())
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_lint/base.py", line 216, in addFailure
    self.callmethod(test_method.autofix_method, failTest)
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_lint/base.py", line 151, in callmethod
    klass_instance.apply(override_origin=test.apply_fix)
  File "/usr/local/lib/python2.7/dist-packages/fontbakery-0.0.14-py2.7.egg/bakery_cli/fixers.py", line 66, in apply
    if not self.fix(*args, **kwargs):
TypeError: fix() takes exactly 3 arguments (1 given)
Error: fix() takes exactly 3 arguments (1 given)
BUILD FAILED
```

**Comment by davelab6** [comment]
That is really weird, because i use `fontbakery-fix-vertical-metrics.py *ttf --autofix` daily and it works fine.

**Comment by felipesanches** [comment]
I agree that this is weird. It was also working for me.

**Comment by davelab6** [comment]
Closing, this will be fixed as a consequence of the 0.1.0 code design
